packages/website/src/components/MentorPerspective.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Button } from '@coderscamp/ui/components/Button';
+import { Center } from '@coderscamp/ui/components/Center';
+import { VStack } from '@coderscamp/ui/components/Stack';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import { SolidDownloadIcon } from '@coderscamp/ui/icons';
+
+export const MentorPerspective = () => {
+  const downloadIcon = useBreakpointValue({ base: undefined, sm: <SolidDownloadIcon /> });
+
+  return (
+    <Center py="80px" backgroundColor="gray.100">
+      <VStack mx={{ base: '30px', xl: 'auto' }} maxW="min(1280px, 100%)">
+        <Typography as="h2" pb="64px" textAlign="center" size="4xl" weight="extrabold" color="gray.900">
```

Comment:
🤔 There should be 32px of space between title and text. You have 64px here + 8px from the `VStack`